src/styles/_catalog.scss

```diff
@@ -1,9 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
+  }
```

Comment:
No need to create a query for the default state.

Reply:
I'll make complete investigation according to this point and surely give a feedback about this point

Reply:
unnecessary query states has been removed